tests/unit/dojson/test_dojson_hepnames.py

```diff
@@ -30,7 +30,10 @@
 from dojson.contrib.marc21.utils import create_record
 
 from inspirehep.dojson.hepnames import hepnames2marc, hepnames
-from inspirehep.dojson.utils import clean_record, get_recid_from_ref
+
+from inspirehep.dojson.utils import clean_record, get_record_ref
+
+from ..conftest import app
```

Comment:
I'm not sure how should I order these imports...

Another thing, in order to be able to import the `app` fixture to use it directly, I have to create `__init__.py` files in the tests dirs, is that ok? (missed them on this commit, if ok will send on next, if not ok, will have to rethink how the app context is created).